RecoVertex/VertexPrimitives/src/CachingVertex.cc

```diff
@@ -288,3 +402,5 @@ CachingVertex<N>::operator TransientVertex() const
 
 template class CachingVertex<5>;
 template class CachingVertex<6>;
+template class CachingVertex<7>;
+template class CachingVertex<8>;
```

Comment:
why are these two needed?
I do not see any code using them.

It also makes me wonder why we need this template argument and several instances